cgsmiles/tests/test_cgsmile_parsing.py

```diff
@@ -3,68 +3,96 @@
 from cgsmiles import read_cgsmiles
 from cgsmiles.read_fragments import strip_bonding_descriptors, fragment_iter
 
-@pytest.mark.parametrize('smile, nodes, edges',(
-                        # smiple linear seqeunce
+@pytest.mark.parametrize('smile, nodes, edges, orders',(
+                        # smiple linear sequence
                         ("{[#PMA][#PEO][#PMA]}",
                         ["PMA", "PEO", "PMA"],
-                        [(0, 1), (1, 2)]),
+                        [(0, 1), (1, 2)],
+                        [1, 1]),
+                        # smiple linear sequenece with multi-edge
+                        ("{[#PMA]1[#PEO]1}",
+                        ["PMA", "PEO"],
+                        [(0, 1)],
+                        [2]),
```

Comment:
```suggestion
                        # simple linear sequence with multi-edge
                        ("{[#PMA]=[#PEO]}",
                        ["PMA", "PEO"],
                        [(0, 1)],
                        [2]),
```
I hate this syntax. Madness lies this way.
Instead, `{[#PMA]=[#PEO]}`.
Keep it closer to smiles, which people know